package.json

```diff
@@ -27,7 +28,7 @@
     "@types/node": "10.12.x",
     "ajv": "6.5.x",
     "chance": "1.0.x",
-    "jest": "23.6.x",
+    "jest": "^23.6.0",
```

Comment:
Please change back to 23.6.x (convention)

Reply:
Not sure how I missed that, good spot!